config/blobs.yml

```diff
@@ -78,30 +78,24 @@ jq-linux64-1.5:
   size: 3027945
   object_id: 41eb1cd1-0377-4e08-5602-51e0af8b9b52
   sha: d8e36831c3c94bb58be34dd544f44a6c6cb88568
-kubernetes-1.11.3/kube-apiserver:
-  size: 185513792
-  object_id: 4c880436-6ded-49a1-4758-99584145da4a
-  sha: 1d92823bcfe71d0c694f219961276405286766be
-kubernetes-1.11.3/kube-controller-manager:
-  size: 154113026
-  object_id: 0d790d64-378a-47cc-7982-1ee222c3809d
-  sha: cddddf92038341053a2ac84a71ce2f147c453604
-kubernetes-1.11.3/kube-proxy:
-  size: 52068695
-  object_id: ecb11b3c-0ef0-4953-5602-6b7547fd4a5d
-  sha: b3c7f41171986bd45eeac5d841cee07b89544888
-kubernetes-1.11.3/kube-scheduler:
-  size: 55636875
-  object_id: 7cfc98ef-6f87-44fe-4cf1-718bf785420e
-  sha: 2a7d78c8354a38add1971dca957d28945eab10d7
-kubernetes-1.11.3/kubectl:
-  size: 55414436
-  object_id: 916cb124-bea1-4428-528d-89da5e4b7758
-  sha: 5a336ea470a0053b1893fda89a538b6197566137
-kubernetes-1.11.3/kubelet:
-  size: 163041864
-  object_id: 39062b5b-80c6-4fe8-4ac3-12c2121f15ad
-  sha: 2c24374244482fc629776366df4f416a1b25ade4
+kubernetes-1.12.2/kube-apiserver:
```

Comment:
The object_id is missing for all blobs; I guess they haven't been uploaded yet?

Reply:
bobs have been uploaded!